src/test/java/org/opensearch/security/TransportUserInjectorIntegTest.java

```diff
@@ -106,11 +106,12 @@ public void testSecurityUserInjection() throws Exception {
             Assert.fail("Expecting exception");
         } catch (OpenSearchSecurityException ex) {
             exception = ex;
-            log.warn(ex.toString());
+            log.debug(ex.toString());
```

Comment:
Warn will always print out. If the test is expected to fail, I was not sure why we would print out the exception message. I swapped to debug since that would be used when we were trying to diagnose unexpected behavior.